README.md

```diff
@@ -69,16 +69,16 @@ const model = {
   // true if the request rejected, and is being called again
   isRetrying: false,
 
-  // start time of request in milliseconds
+  // start time of request in milliseconds UTC
```

Comment:
For this to be true, do we not also have to modify the `getTime` function to be `(new Date().utc()).getTime()`